### ISSUE TITLE
[Accessibility] TalkBack does not announce Pivot tabs position

### ISSUE THREAD

**Issue by lutorre** [microsoft/fluentui#18854]
### Environment Information

- **Package version(s)**: Android 11
- **Browser and OS versions**:  Google chrome

### Describe the issue:

**Steps to reproduce:**

Open chrome browser and open a web page that contains a Pivot component with two or more tabs.

Link to component: https://fabricweb.z5.web.core.windows.net/oufr/8.22.1/#/examples/pivot

#### Actual behavior:

Talkback announces only the tab title

#### Expected behavior:

TalkBack announces the tan title followed by the tab position (example, 'Tab one, 1 of two')

**Comment by smhigley** [comment]
Talkback doesn't announce position for any tabs -- you can see this by running it on the ARIA Practices tabs: https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html

It also doesn't announce "X of Y" for native tabs, if you check on native apps with tabs :)
